data/CMakeLists.txt

```diff
@@ -48,11 +47,21 @@ add_custom_command(OUTPUT ${PROJ_DB}
 
 add_custom_target(generate_proj_db ALL DEPENDS ${PROJ_DB})
 
+if(NOT "${CMAKE_CURRENT_SOURCE_DIR}" STREQUAL "${CMAKE_CURRENT_BINARY_DIR}")
+    foreach(FILE ${PROJ_DICTIONARY}
+                 null
```

Comment:
BTW if `null` is expected be installed, it must be added to `PROJ_DICTIONARY` instead.

Reply:
yes, null should be installed. It is is by automake builds